configure.ac

```diff
@@ -1310,9 +1310,23 @@ elif test x$with_runtime_preset = xwinaot; then
    mono_feature_disable_reflection_emit='yes'
    mono_feature_disable_appdomains='yes'
 
-   AOT_BUILD_FLAGS="--runtime=mobile --aot=full,$INVARIANT_AOT_OPTIONS"
+   AOT_BUILD_FLAGS="--runtime=mobile -O=gsharedvt --aot=full,$INVARIANT_AOT_OPTIONS"
```

Comment:
That's surprising you need to pass that with `--aot=full`.

Reply:
Just did the same as we had on the other full AOT builds, it should still be the default optimization for full AOT, so probably not needed on any of the full AOT test builds.